### PR TITLE
feat: add adopters page

### PR DESCRIPTION
**Pull request by asmitbm**
This page contains a list of projects leveraging ORAS, as well as registries that are known to support [OCI Artifacts](https://github.com/opencontainers/artifacts).

Edit: Adopters can submit an issue via the [issue template](https://github.com/oras-project/oras-www/blob/38cb726e1f70b58bfcfc2f3b07cbd62fbda52c70/.github/ISSUE_TEMPLATE/submit_adopter.yml) and then we can verify the details and add them to the list.

### CODE REVIEW

**Comment by asmitbm** [comment]
Cc: @FeynmanZhou 

The adopters page is almost complete, only the content part of this page is remaining. After that, this PR will be ready for review.

File: static/adopters/data.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
+const projects = [
```
Comment (FeynmanZhou) [inline static/adopters/data.js:1]: Could you double check the project intro on each card? For example, the Helm project intro is not correct
image

**Review by FeynmanZhou — CHANGES_REQUESTED** [review]
PTAL

**Comment by asmitbm** [comment]
> PTAL

Actually Helm is just mentioned as package manager for Kubernetes, hence I researched more and add that info. Will change it to product docs now.

**Review by FeynmanZhou — APPROVED** [review]
LGTM

**Pull request merged**